content/api/integrations/aws.md

```diff
@@ -27,4 +27,18 @@ Configure your Datadog-AWS integration directly through Datadog API.
     [Read more about EC2 tagging in AWS tagging documentation](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html).
 
 * `host_tags` [*optional*, *default*=**None**]:  
-    Array of tags (in the form `key:value`) to add to all hosts and metrics reporting through this integration.
\ No newline at end of file
+    Array of tags (in the form `key:value`) to add to all hosts and metrics reporting through this integration.
+
+* `run_check` [*optional*, *default*=**false**]:  
+    Determines if the integration install check is run before returning a response.
+
+    * If **true**:
+
+        - The install check is run
+        - If there’s an error in the configuration the error is returned
+        - If there’s no error, *204 No Content* response code is returned
+
+    * If **false**:
+
+        - We return a *202 accepted*
+        - Install check is run after returning a response
```

Comment:
`run_check` is currently only for pagerduty, slack, and webhooks.

Reply:
👍